Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -69,6 +70,18 @@ class RouterSpec: QuickSpec {
                     expect(router.process(request: anOptionsRequest)).to(equal(expectedResponse))
                 }
             }
+            
+            context("when receive an unknown request to an existing resource") {
```

Comment:
This test is testing too much for a unit test. (Although it is a reasonable function/higher-level test).

What is the failure message when they are not equal? What are some ways this test could be split?

Reply:
will look at splitting up this test and make it more granular

Reply:
will look into that

Reply:
all unit tests are further broken down